components/Header.css

```diff
@@ -0,0 +1,46 @@
+.navi {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 1rem 0;
+	padding: 1rem 2rem;
+	background: var(--box-shadow);
+border-radius: 0.4rem;
+}
+
+.navi h2{
+	font-size: 3rem;
+}
+.navi ul {
+	display: flex;
+	list-style: none;
+}
+
+.navi li {
+	padding-left: 1rem;
+}
+
+.navi a {
+	text-decoration: none;
+	color: var(--primary-color);
+	background-color: var(--white-color);
```

Comment:
It's not recommended to use color name as variable name. Else everything looks good to me!!